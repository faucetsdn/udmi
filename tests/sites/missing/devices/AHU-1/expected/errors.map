Exceptions for AHU-1
  While loading credentials for local device AHU-1
    Found 0 credentials
  1 schema violations found
    /system/location/site: ECMA 262 regex "^[A-Z]{2}-[A-Z]{3}-[A-Z0-9]{2,9}$" does not match input string "ZZ-TRI-FECTA-AHU"
  expected files
    Missing files: [rsa_private.pem, rsa_private.pkcs8, rsa_public.pem]
