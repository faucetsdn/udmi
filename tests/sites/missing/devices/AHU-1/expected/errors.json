Exceptions for AHU-1
  While loading credentials for local device AHU-1
    Found 0 credentials
  1 schema violations found
    /system/location/site: ECMA 262 regex "^[A-Z]{2}-[A-Z]{3}-[A-Z0-9]{2,9}$" does not match input string "ZZ-TRI-FECTA-AHU"
  devices/AHU-1
    Missing files: [rsa_private.pem, rsa_private.pkcs8, rsa_public.pem]
